### PR TITLE
List

### PR DESCRIPTION
**Pull request by SiansandaTamara**
##implemented the following
-Add webpack file
-Add linter packages
-Add git flow
-Use of html/css.js
-Add button
-update reademe file
-Active the add and remove button"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
-# To-Do
\ No newline at end of file
+# To-Do
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This a group work project testing basic Javascript skills.
+
+# WORKFLOW SCREENSHOT
+![screenshot]()
```
Comment (MahmoudBakr23) [inline README.md]: - [ ] It seems you forgot to edit the standard empty README file. Please make sure to do so and add a valid image of the app.

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+- HTML
+- CSS
+- JavaScript
+## Live Demo (if available)
+
+[Live Demo Link]
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
```
Comment (MahmoudBakr23) [inline README.md]: - [ ] You should add correct structures to install the app as well.

File: README.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+👤 **Tamara Siansanda**
+
+- GitHub: [@SiansandaTamara](https://github.com/SiansandaTamara)
+- Twitter: [@twitterhandle](https://twitter.com/TamaraSiansanda)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (MahmoudBakr23) [inline README.md]: - [ ] Please fix the issues page link.

File: README.md [hunk 4]
```diff
@@ -1 +1,59 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (MahmoudBakr23) [inline README.md]: - [ ] Please fix the MIT link.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @SiansandaTamara 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done implementing the add, remove and drag functions
- Correct Linters
- Descriptive PR

**Optional**
- Kindly make sure to do the below comments for a professional README.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SiansandaTamara** [comment]
Thank you I 'll do the needful it's only that I didn't sleep the whole night trying to work on it.Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Mahmoud Bakr ***@***.***> Date: 05/05/2022  04:52  (GMT+02:00) To: SiansandaTamara/To-Do ***@***.***> Cc: tamara siansanda ***@***.***>, Mention ***@***.***> Subject: Re: [SiansandaTamara/To-Do] List (PR #1) 

@MahmoudBakr23 approved this pull request.



Hi @SiansandaTamara 😃,

Your project is complete!  There is nothing else to say other than... it's time to merge it 

Congratulations! 🎉

Highlights



Well done implementing the add, remove and drag functions

Correct Linters

Descriptive PR



Optional



Kindly make sure to do the below comments for a professional README.



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.



In README.md:

> \ No newline at end of file

+# To-Do

+

+

+

+> This a group work project testing basic Javascript skills.

+

+# WORKFLOW SCREENSHOT

+screenshot





 It seems you forgot to edit the standard empty README file. Please make sure to do so and add a valid image of the app.









In README.md:

> +

+To get a local copy up and running follow these simple example steps.





 You should add correct structures to install the app as well.









In README.md:

> +```bash

+git clone// https://github.com/SiansandaTamara/todo

+

+

+

+## Authors

+👤 **Tamara Siansanda**

+

+- GitHub: ***@***.***(https://github.com/SiansandaTamara)

+- Twitter: ***@***.***(https://twitter.com/TamaraSiansanda)

+

+## 🤝 Contributing

+

+Contributions, issues, and feature requests are welcome!

+

+Feel free to check the [issues page](../../issues/).





 Please fix the issues page link.









In README.md:

> +

+Feel free to check the [issues page](../../issues/).

+

+## Show your support

+

+Give a ⭐️ if you like this project!

+

+## Acknowledgments

+

+- Hat tip to anyone whose code was used

+- Inspiration

+- etc

+

+## 📝 License

+

+This project is [MIT](./MIT.md) licensed.





 Please fix the MIT link.





—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

[

{

***@***.***": "http://schema.org",

***@***.***": "EmailMessage",

"potentialAction": {

***@***.***": "ViewAction",

"target": "https://github.com/SiansandaTamara/To-Do/pull/1#pullrequestreview-962814807",

"url": "https://github.com/SiansandaTamara/To-Do/pull/1#pullrequestreview-962814807",

"name": "View Pull Request"

},

"description": "View this Pull Request on GitHub",

"publisher": {

***@***.***": "Organization",

"name": "GitHub",

"url": "https://github.com"

}

}

]

File: package.json [hunk 5]
```diff
@@ -0,0 +1,629 @@
+{
```
Comment (Meltrust) [inline package.json:1]: - According to the project requirements:

image

Therefore:

- [ ] Please, ensure that:

- Basic commands for building Webpack in development mode, production mode is configured
- Webpack dev server is set up and running with no errors.

## Explanation

This project should not be run with the "live server" VsCode extension.  This project should be run with the command "npm start".  Also, there should be a way to run the project in "watch" and "production" modes.  You will need to configure webpack and package.json for this.

## Hints

- Add this "scripts" section to your package.json:

image

- Install the webpack dev server, CLI, Html plugin, and CSS loader:

```bash
npm install webpack-dev-server --save-dev
npm install -D webpack-cli
npm install --save-dev style-loader
npm install css-loader -D
npm install html-webpack-plugin
```

- Use this webpack config:

```javascript
const path = require('path');
const HtmlWebpackPlugin = require('html-webpack-plugin');

module.exports = {
  mode: 'development',
  entry: './src/index.js',
  devtool: 'inline-source-map',
  devServer: {
    static: './dist',
  },
  plugins: [
    new HtmlWebpackPlugin({
      template: './src/index.html',
    }),
  ],
  output: {
    filename: 'main.js',
    path: path.resolve(__dirname, 'dist'),
  },
  module: {
    rules: [
      {
        test: /\.css$/i,
        use: ['style-loader', 'css-loader'],
      },
    ],
  },
};
```
- [ ] OPTIONAL: I highly suggest getting rid of the hundreds of unnecessary dependencies your project has.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @SiansandaTamara,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice styling!✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
Hi @SiansandaTamara 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by andersonlebon** [comment]
[ OPTIONAL ] I suggest you add in your` README.md` file commands on how to install dependencies and run the app.

**Pull request merged**